config/config.go

```diff
@@ -143,7 +143,13 @@ func (nc *NodeConfig) GetViperConfig(v *viper.Viper) error {
 // This function is called in cosmos-sdk.
 func AddFlags(cmd *cobra.Command) {
 	def := DefaultNodeConfig
-	cmd.Flags().Bool(FlagAggregator, def.Aggregator, "run node in aggregator mode")
+
+	// special case for aggregator flag
+	// there are 3 options: true, false, and empty (which means true)
+	aggregator := NewFlexibleBool(def.Aggregator)
```

Comment:
why is this simple change won't work?
`cmd.Flags().BoolVar(&def.Aggregator, FlagAggregator, true, "run node in aggregator mode")`
and then if you dont pass anything, it will be `true`, if you want to unset it, you can call `rollkit start --rollkit.aggregator=false`

Reply:
☝🏻

Reply:
I think we should keep the logic where we set it to true explicitly in the run command.
But trying it against my tests show that BoolVar works with "=".